Adding winui3 sample, separate SLN due to project structure needs (#142)
MicrosoftEdge/WebView2Samples: SampleApps/WebView2_WinUI3_Sample/readme.md

> # Introduction

Reviewer: Add YAML fields, and onboard this sample so it appears at https://docs.microsoft.com/en-us/samples/browse/?terms=webview2

will msg offline how to do this

Reviewer: Onboarding the sample is independent from merging this PR.  Closing thread.  Entered a Comment/Suggestion that provides the YAML fields.

> # Introduction

Reviewer: ```suggestion
# WebView2_WinUI3_Sample

> This sample shows off using a WebView2 control in a WinUi 3 Windows SDK Packaged application.

Reviewer: Global: use spelling "WinUI 3" except in code.  per https://docs.microsoft.com/en-us/windows/apps/winui/winui3/
```suggestion
This sample shows off using a WebView2 control in a WinUI 3 Windows SDK Packaged application.
```

> It also optionaly shows how you would update the application to ship with a fixed WebView2 version instead of using the version installed and running on the Windows computer.

Reviewer: ```suggestion
It also optionaly shows how you would update the application to ship with a fixed version of the WebView2 Runtime instead of using the version that's installed and running on the user's computer.
```

# Relevant directories

| Directory | Contents |
--- | --- |
| WebView2_WinUI3_Sample | Project code |
| WebView2_WinUI3_Sample (Package) | Packaging and distribution project |
| WebView2_WinUI3_Sample (Package)\FixedRuntime | (Optional) Fixed WebView2 runtime |
| WebView2_WinUI3_Sample (Package)\FixedRuntime\95.0.1020.53 | (Optional) Fixed WebView2 runtime sample |

# Fixed version usage

> If you want to ship a fixed version of the WebView2 runtime with your application you will need to include it in your project.

Reviewer: ```suggestion
If you want to ship a fixed version of the WebView2 Runtime packaged with your application, you will need to include the WebView2 Runtime in your project.
```

> Instructions can be found at: https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution

Reviewer: ```suggestion
Instructions can be found at: https://docs.microsoft.com/microsoft-edge/webview2/concepts/distribution
```

> The following assume you are using runtime version 95.0.1020.53, you will change this number to whatever version you are using.

Reviewer: ```suggestion
The following instructions assume you are using WebView2 Runtime version 95.0.1020.53.  You'll change this number to whatever version you are using.
```

You will need to:

> 1 Include the fixed WebView2 runtime in the package project

Reviewer: ```suggestion
1. Include the fixed version of the WebView2 Runtime in the package project:

```

> <pre>\WebView2_WinUI3_Sample\WebView2_WinUI3_Sample (Package)\FixedRuntime\95.0.1020.53\</pre>

Reviewer: ```suggestion
`\WebView2_WinUI3_Sample\WebView2_WinUI3_Sample (Package)\FixedRuntime\95.0.1020.53\`
```

> 2 Update the package project wapproj file for the version your using

Reviewer: ```suggestion

2. Update the package project `wapproj` file for the version you're using:

```

> <pre> < Content Include="FixedRuntime\95.0.1020.53\\**\*.*" > </pre>

Reviewer: ```suggestion

   `< Content Include="FixedRuntime\95.0.1020.53\\**\*.*" >`

```

> 3 Uncomment the code in app.xaml.cs to enable the runtime override

Reviewer: ```suggestion
3. Uncomment the code in `app.xaml.cs` to enable the runtime override:
```

> <pre>

Reviewer: I don't think this can be tagged in this Suggestion UI.  Recomm: 

Above start of code block, put 3 backticks (indented 3 or 4 spaces, to match "U" of "Uncomment").  To right of triple backticks, add a lang abbrev such as csharp (no delim space).  

Below end of code block, put 3 backticks (indented 3 or 4 spaces consistent w/ the containing list in this Markdown file).

Indent lines of code 3 or 4 spaces, consistent w/ the containing list.

public App()
{
    this.InitializeComponent();
    // If you are shipping a fixed version WebView2 SDK with your application you will need

>     // to use the following code (update the runtime version to what you are shipping.

Reviewer: ```suggestion
    // to use the following code (update the runtime version to what you are shipping).
```

    StorageFolder localFolder = Windows.ApplicationModel.Package.Current.InstalledLocation;
    String fixedPath = Path.Combine(localFolder.Path, "FixedRuntime\\95.0.1020.53");
    Debug.WriteLine($"Launch path [{localFolder.Path}]");
    Debug.WriteLine($"FixedRuntime path [{fixedPath}]");
    Environment.SetEnvironmentVariable("WEBVIEW2_BROWSER_EXECUTABLE_FOLDER", fixedPath);
}
</pre>

> 4 Update the version information for the appropriate version

Reviewer: ```suggestion

4. Update the version information for the appropriate version:

```

Reviewer: or use "1. " for every numbered step, as Docs do

> 3 Uncomment the code in app.xaml.cs to enable the runtime override
> <pre>
> public App()
> {
>     this.InitializeComponent();
>     // If you are shipping a fixed version WebView2 SDK with your application you will need
>     // to use the following code (update the runtime version to what you are shipping.
>     StorageFolder localFolder = Windows.ApplicationModel.Package.Current.InstalledLocation;
>     String fixedPath = Path.Combine(localFolder.Path, "FixedRuntime\\95.0.1020.53");
>     Debug.WriteLine($"Launch path [{localFolder.Path}]");
>     Debug.WriteLine($"FixedRuntime path [{fixedPath}]");
>     Environment.SetEnvironmentVariable("WEBVIEW2_BROWSER_EXECUTABLE_FOLDER", fixedPath);
> }
> </pre>
> 4 Update the version information for the appropriate version
> <pre> 
> String fixedPath = Path.Combine(localFolder.Path, "FixedRuntime\\95.0.1020.53");
> </pre>

Reviewer: ```suggestion
3. Uncomment the code in `app.xaml.cs` to enable the runtime override:

   ```
   public App()
   {
       this.InitializeComponent();
       // If you are shipping a fixed version WebView2 SDK with your application you will need
       // to use the following code (update the runtime version to what you are shipping.
       StorageFolder localFolder = Windows.ApplicationModel.Package.Current.InstalledLocation;
       String fixedPath = Path.Combine(localFolder.Path, "FixedRuntime\\95.0.1020.53");
       Debug.WriteLine($"Launch path [{localFolder.Path}]");
       Debug.WriteLine($"FixedRuntime path [{fixedPath}]");
       Environment.SetEnvironmentVariable("WEBVIEW2_BROWSER_EXECUTABLE_FOLDER", fixedPath);
   }
   ```

4. Update the version information for the appropriate version:

   ```
   String fixedPath = Path.Combine(localFolder.Path, "FixedRuntime\\95.0.1020.53");
   ```
```

> <pre> 
> String fixedPath = Path.Combine(localFolder.Path, "FixedRuntime\\95.0.1020.53");
> </pre>

Reviewer: ```suggestion
   ```
String fixedPath = Path.Combine(localFolder.Path, "FixedRuntime\\95.0.1020.53");
   ```
```

> # WebView2_WinUI3_Sample

Reviewer: Not ready to Commit yet; draft/starter:
```suggestion
---
page_type: sample
languages:
- csharp
- fsharp
- vbnet
products:
- office
- windows
- xbox
---
# WebView2_WinUI3_Sample
```

Reviewer: @DavidShoe - Is the list of coding "languages: " above correct?

This sample shows off using a WebView2 control in a WinUI 3 Windows SDK Packaged application.

This sample also shows how to optionally update the application to ship packaged together with a fixed version of the WebView2 Runtime, instead of using the version of the WebView2 Runtime that's installed on the user's computer.

> 

Reviewer: Add info like top of other samples' readmes: 
This project was created using Visual Studio 2022[|2019].  It uses [programming language list] in the WebView2 context.

motivation for PR comment: I just opened this project in VS2019 but I think I should reopen it in VS2022, need a hint here.
